Xwt/Xwt.Backends/IWindowFrameBackend.cs

```diff
@@ -82,10 +82,22 @@ public interface IWindowFrameBackend: IBackend
 		bool Close ();
 
 		/// <summary>
-		/// Gets or sets a value indicating whether this window is in full screen mode
+		/// Gets or sets a value indicating whether this window is iconified (true) or normal (false).
+		/// </summary>
+		/// <value><c>true</c> if the window is iconified; otherwise, <c>false</c>.</value>
+		bool Iconify { get; set; }
+
+		/// <summary>
+		/// Gets or sets a value indicating whether this window is in full screen mode (true) and normal (false).
 		/// </summary>
 		/// <value><c>true</c> if the window is in full screen mode; otherwise, <c>false</c>.</value>
 		bool FullScreen { get; set; }
```

Comment:
If we add the WindowState property, the Iconify and FullScreen properties are redundant in the backend, they can be removed.